repository_templates/java/.github/workflows/java-ci.yaml

```diff
@@ -16,17 +16,21 @@ jobs:
         working-directory: ./java
 
     steps:
+      - uses: actions/checkout@v4
+
       - name: Use Java SDK {{ `${{ env.JAVA_VERSION }}` }}
         uses: actions/setup-java@v4
         with:
           java-version: {{ `${{ env.JAVA_VERSION }}` }}
           distribution: 'temurin'
           check-latest: true
 
-      - name: Test and build Jar
+      - name: Java setup Gradle
         uses: gradle/actions/setup-gradle@v3
         env:
           CHECK_GENERATED_FILES: true
         with:
           gradle-version: '8.8'
+
+      - name: Java build
```

Comment:
The env. CHECK_GENERATED_FILES should be in this step also, right?

Reply:
I was checking this env variable and I saw that it isn't used in our project. I followed how it was done in other Grafana project but it has a lot of things deprecated.

I also updated the release CI workflow.